### PR TITLE
Readme contribution guide

### PR DESCRIPTION
**Pull request by KTGypsy**
## What:

* Extended readme file with examples.

## Why:

* For easier integration in projects. By devs for devs.

## Example:

* Look at the README.md file

### CODE REVIEW

**Review by PaulinaSadowskaAllegro — DISMISSED** [review]

File: README.md [hunk 1]
```diff
@@ -1,8 +1,196 @@
...
+    implementation "pl.allegro.mobile:json-logic-core-jvm:1.0.0"
+    implementation "pl.allegro.mobile:json-logic-operations-stdlib-jvm:1.0.0"
+    // KMP target
+    implementation "pl.allegro.mobile:json-logic-core:1.0.0"
+    implementation "pl.allegro.mobile:json-logic-operations-stdlib:1.0.0"
+}
+```
+with CocoaPods:
+
+To integrate **JsonLogicKMP** into your Xcode project using CocoaPods, specify it in your Podfile:
```
Comment (lsolniczek) [inline README.md:25]:
```suggestion
To integrate **JsonLogicKMP** with your Xcode project using CocoaPods, specify it in your Podfile:
```

File: README.md [hunk 2]
```diff
@@ -1,8 +1,196 @@
...
+        .Builder()
+        .addLogger(jsonLogicLoggingCallback)
+        .build()    
+}
+```
+
+and Swift:
+
+```swift
+    let logger = Logger(subsystem: Bundle.moduleBundle.bundleIdentifier ?? Bundle.moduleName, category: Self.loggerCategory)
```
Comment (lsolniczek) [inline README.md:53]: What is `Self.loggerCategory`?

File: README.md [hunk 3]
```diff
@@ -1,8 +1,196 @@
...
+            .build()
+    }
+```
+
+Json logic expressions evaluation
+-------------------
+
+Logic expressions are represented by maps where key is operation name and value is arguments. User might provide some additional input using **data** parameter. 
+
+Example expression evaluation in Kotlin is shown below:
```
Comment (lsolniczek) [inline README.md:95]:
```suggestion
Example of expression evaluation in Kotlin is shown below:
```

File: README.md [hunk 4]
```diff
@@ -1,8 +1,196 @@
...
+func evaluateAdditionOperation() -> JsonLogicResult {
+    let engine = JsonLogicKMP.JsonLogicEngineBuilder().build()
+    let logicExpression: [String: Any] = ["+": [1, ["var": "second"]]]
+    let logicData: [String: Any] = ["second": 2]
+        
+    return engine.evaluate(expression: logicExpression, data: logicData)
+}
+```
+
+Expressions evaluation results in **JsonLogicResult**. It indicates if evaluation is successful or what kind of error occured.
```
Comment (lsolniczek) [inline README.md:119]:
```suggestion
Expressions evaluation results is **JsonLogicResult**. It indicates if evaluation is successful or what kind of error occured.
```

**Review by lsolniczek — CHANGES_REQUESTED** [review]

**Review by lsolniczek — APPROVED** [review]

**Review by PaulinaSadowskaAllegro — APPROVED** [review]

**Pull request merged**